buildscripts/kokoro/unix.sh

```diff
@@ -50,6 +50,9 @@ export LD_LIBRARY_PATH=/tmp/protobuf/lib
 export LDFLAGS=-L/tmp/protobuf/lib
 export CXXFLAGS="-I/tmp/protobuf/include"
 
+# Warm mvn cache with retries, in case connection to maven repo is flakey
+(./gradlew assemble || ./gradlew assemble || ./gradlew assemble) && ./gradlew clean
```

Comment:
How about `./gradlew -m build` (`-m` is "dry run")? That skips the actual tasks but appears like it downloads necessary deps.